### PR TITLE
[BugFix] Fix worker scaling management

### PR DESCRIPTION
**Pull request by matthew-so**
This pull request fixes the sole remaining issue(s) with worker scaling management through the EvalAI UI. To date, these are:

1. The UI not properly displaying the current worker configuration

### CODE REVIEW

File: apps/challenges/aws_utils.py [hunk 1]
```diff
@@ -664,6 +664,7 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
             challenge.task_def_arn = None
             challenge.save()
         else:
+            response["Error"] = "Scaling worker not supported for inactive worker. Full error:\n" + response["Error"]
```
Comment (Ram81) [inline apps/challenges/aws_utils.py:667]: Change message to `Scaling inactive workers not supported`

File: frontend/src/js/controllers/challengeCtrl.js [hunk 2]
```diff
@@ -1639,7 +1639,7 @@
             };
             parameters.callback = {
                 onSuccess: function() {
-                    $rootScope.notify("success", "Evaluation worker resources scaled successfully");
+                    $rootScope.notify("success", "Evaluation worker resources scaled successfully. Please wait until the old worker is stopped and the new worker starts before making further changes.");
```
Comment (Ram81) [inline frontend/src/js/controllers/challengeCtrl.js:1642]: Change message to `Evaluation worker resources scaled successfully.`

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: apps/challenges/aws_utils.py (hunk 1 above)
Comment (matthew-so) [reply apps/challenges/aws_utils.py]: Done

File: frontend/src/js/controllers/challengeCtrl.js (hunk 2 above)
Comment (matthew-so) [reply frontend/src/js/controllers/challengeCtrl.js]: Done

**Review by Ram81 — APPROVED** [review]

**Pull request merged**